scopesim/system_dict.py

```diff
@@ -116,6 +127,52 @@ def __str__(self) -> str:
         return output
 
 
+class UniqueList(MutableSequence):
+    """Ordered collection with unique elements."""
```

Comment:
`UniqueList` seems a bit overkill, maybe it would be cleaner to just have a non-unique list. But it isn't really a problem either, so let's keep it.